Unity-Technologies/ScriptableRenderPipeline
# Fix FB#1131211 - Voronoi Node Serialization Issue

### Purpose of this PR
Why is this PR needed, what hard problem is it solving/fixing?
During a recent refactor `VoronoiNode` class got renamed in error, causing all previously serialized voronoi nodes fail to deserialize. 

This PR fixes the name of the class, while maintaining the ability to deserialize previously serialized node instances (from both the previous broken versions, and the ones serialized with the wrong names).

---
### Release Notes
Please add any useful notes about the feature/fix that might be helpful for others.
Changelog

---
### Testing status
**Katana Tests**: First off we need to make sure the Katana SRP tests are green?
Pass

**Manual Tests**: What did you do?
- Other: 

**Automated Tests**: What did you setup?
This fixes a previously broken automated test (which is in PR #2845 )

Any test projects to go with this to help reviewers?
See bug case

---
### Overall Product Risks
**Technical Risk**: None, Low, Medium, High? None

**Halo Effect**: None, Low, Medium, High? Low - Affects serialization, but fixes it.

---
### Comments to reviewers
Notes for the reviewers you have assigned.
